pkg/printers/internalversion/printers_test.go

```diff
@@ -2987,44 +2920,6 @@ func TestPrintPodDisruptionBudget(t *testing.T) {
 	}
 }
 
-func TestAllowMissingKeys(t *testing.T) {
-	tests := []struct {
-		Name      string
-		PrintOpts *printers.PrintOptions
-		Input     runtime.Object
-		Expect    string
-		Error     string
-	}{
-		{"test template, allow missing keys", &printers.PrintOptions{OutputFormatType: "template", OutputFormatArgument: "{{.blarg}}", AllowMissingKeys: true}, &api.Pod{}, "<no value>", ""},
```

Comment:
Link me to the replacement test on the new/current printers please.

Reply:
https://github.com/kubernetes/kubernetes/blob/master/pkg/printers/template_flags_test.go#L140